src/pages/EditMonitor.vue

```diff
@@ -1264,11 +1316,27 @@ message HealthCheckResponse {
                     this.monitor.port = "53";
                 } else if (this.monitor.type === "radius") {
                     this.monitor.port = "1812";
+                } else if (this.monitor.type === "snmp") {
+                    this.monitor.port = "161";
                 } else {
                     this.monitor.port = undefined;
                 }
             }
 
+            // Set a default timeout of 1 second for SNMP monitors when querying a single OID.
+            // Since we're only querying a single OID, a shorter timeout is sufficient to ensure timely responses
+            // without unnecessary delays. This helps keep the monitoring process lightweight and efficient.
+            if (this.monitor.type === "snmp") {
+                this.monitor.timeout = 1;
```

Comment:
https://www.arubanetworks.com/techdocs/AirWave/8301/webhelp/Content/BestPractices/Chapter2/Setting_Up_Recommended_T.htm has the at a min of 3 => lets increase the default to 5

Also I have tried to reformulate what I think you are trying to convery here

```suggestion
            if (this.monitor.type === "snmp") {
                // snmp is not expected to be executed via the internet => we can choose a lower default timeout
                this.monitor.timeout = 5;
```